src/pkg/packaging/deb/package.targets

```diff
@@ -90,6 +100,12 @@
       <SharedHostTokenValue Include="%SHARED_HOST_BRAND_NAME%">
         <ReplacementString>$(SharedHostBrandName)</ReplacementString>
       </SharedHostTokenValue>
+     <SharedHostTokenValue Include="%RUNTIME_DEPS_DEBIAN_PACKAGE_NAME%">
+        <ReplacementString>$(RuntimeDependenciesDebPkgName)</ReplacementString>
+      </SharedHostTokenValue>
+      <SharedHostTokenValue Include="%RUNTIME_DEPS_VERSION%">
+        <ReplacementString>$(SharedFrameworkNugetVersion)</ReplacementString>
```

Comment:
`SharedFrameworkNugetVersion` seems like the wrong variable here.  It should only be used when actually referencing the SharedFramework.  Here we are referencing the runtime-deps debian package.

I think this justifies its own new property, whose value get defaulted to `$(ProductVersion)`.